charts/gha-runner-scale-set/templates/_helpers.tpl

```diff
@@ -476,25 +476,46 @@ volumeMounts:
   {{- end }}
 {{- end }}
 {{- if eq $searchControllerDeployment 1 }}
-  {{- $counter := 0 }}
+  {{- $multiNamespacesCounter := 0 }}
+  {{- $singleNamespaceCounter := 0 }}
   {{- $controllerDeployment := dict }}
+  {{- $singleNamespaceControllerDeployments := dict }}
   {{- $managerServiceAccountName := "" }}
   {{- range $index, $deployment := (lookup "apps/v1" "Deployment" "" "").items }}
-    {{- range $key, $val := $deployment.metadata.labels }}
-      {{- if and (eq $key "app.kubernetes.io/part-of") (eq $val "gha-runner-scale-set-controller") }}
-        {{- $counter = add $counter 1 }}
-        {{- $controllerDeployment = $deployment }}
+    {{- if kindIs "map" $deployment.metadata.labels }}
+      {{- if eq (get $deployment.metadata.labels "app.kubernetes.io/part-of") "gha-runner-scale-set-controller" }}
```

Comment:
Is this a new value that we should set for the part-of?
The [ADR](https://github.com/actions/actions-runner-controller/blob/master/adrs/2022-12-05-adding-labels-k8s-resources.md#controller-manager) says just "actions-runner-controller"
Should we update the ADR?

Reply:
Good shout, the ADR pre dates the rename.